NativeAuthSampleApp.xcodeproj/project.pbxproj

```diff
@@ -470,7 +470,7 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/AzureAD/microsoft-authentication-library-for-objc";
 			requirement = {
-				kind = upToNextMajorVersion;
+				kind = upToNextMinorVersion;
 				minimumVersion = 1.3.0;
```

Comment:
Can you specify 1.3.1 as minimum version?

Reply:
Done